lib/tasks/gettext.rake

```diff
@@ -52,6 +52,68 @@ namespace :gettext do
            files_to_translate.join(' '))
   end
 
+  def metadata_pot_file_path
+    File.join(locale_path, GettextSetup.config['project_name'] + '_metadata.pot')
+  end
+
+  def generate_new_pot_metadata
+    open(metadata_pot_file_path, 'w') do |f|
+      f << "#\n"
```

Comment:
Can we just use a HEREDOC for all of these? For example:

```
f << <<-DOC
#
#, fuzzy
# msgid ""
# msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
...
DOC
```

That way it's more like editing an actual file whenever we change this.